README.md

```diff
@@ -42,8 +42,28 @@ end
 
 Details for adding the handler to:
 
-- [Lucky Framework](https://luckyframework.org/guides/http-and-routing/http-handlers)
-- [Amber Framework](https://docs.amberframework.org/amber/guides/routing/pipelines#sharing-pipelines)
+##### Reporting errors in [Lucky Framework](https://luckyframework.org)
+
+1. Add the shard to `shard.yml`
+2. Require the shard in `src/shards.cr`
+3. Add the built-in `Honeybadger::AuthenticHandler` to your middleware stack:
+
+    ```crystal
+    def middleware : Array(HTTP::Handler)
+      [
+        # ...
+        Lucky::ErrorHandler.new(action: Errors::Show),
+        Honeybadger::AuthenticMiddleware.new,
```

Comment:
I think this is just a late stage rename on your part, but below you have `Honeybadger::AuthenticHandler` instead of _middleware_